src/Clip.cpp

```diff
@@ -308,46 +320,71 @@ float Clip::End() const
 		return end;
 }
 
-// Get an openshot::Frame object for a specific frame number of this reader.
-std::shared_ptr<Frame> Clip::GetFrame(int64_t requested_frame)
+// Create an openshot::Frame object for a specific frame number of this reader.
+std::shared_ptr<Frame> Clip::GetFrame(int64_t frame_number)
+{
+	// Check for open reader (or throw exception)
+	if (!is_open)
+		throw ReaderClosed("The Clip is closed.  Call Open() before calling this method", "N/A");
```

Comment:
```suggestion
		throw ReaderClosed("The Clip is closed.  Call Open() before calling this method.");
```
The `file_path` argument to `ReaderClosed` is optional.